### PR TITLE
799, add light header option to table

### PR DESCRIPTION
**Pull request by rsadres**
## PR Checklist

- Manually tested across supported browsers
  - [ ] Chrome
  - [ ] Firefox
  - [ ] Safari
  - [ ] Edge (Win 10)
- [ ] Unit tests written (`common` at minimum)
- [ ] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [x] One core team UX approval

### CODE REVIEW

**Comment by sodiumjoe** [comment]
UX wants this to be the default, but since that would be a breaking change we put it behind a top-level flag on `Table`. Next major I think we should reverse it so the dark header option is behind a flag.

**Comment by chrisdopuch** [comment]
@rsadres looks like the Travis CI build failed, you'll want to look into that before merging
https://travis-ci.org/appnexus/lucid/builds/268072188?utm_source=github_status&utm_medium=notification
I think it just looks like some out of date jest snapshots

File: src/components/Table/Table.less [hunk 1]
```diff
@@ -49,13 +49,55 @@
 	}
 
 	// Table header styles
+	&.lucid-Table-has-light-header > &-Thead {
```
Comment (chrisdopuch) [inline src/components/Table/Table.less]: it seems like there's a lot of repetition of styles not strictly related to the light-header. Possible to collapse this with the non-light-header table styles?

Comment (rsadres) [inline src/components/Table/Table.less]: I don't know :) I'll look at it together with one of you

**Pull request merged**